offline/ci.sh

```diff
@@ -101,16 +116,16 @@ calling_charts=(
 )
 
 # wire_version=$(helm show chart wire/wire-server | yq -r .version)
-wire_version="4.39.0"
+wire_version="4.39.159"
```

Comment:
Hm, so the version that is currently on staging will be the version that our users should install? Was this an active choice by someone? :thinking: 

Or, will this branch not be merged to `master`?

Reply:
there were changes needed in the wire-server helm chart to support k8s 1.28
This change is temporary, goal is to replace it with "4.40" as soon as that is released and than merge